topydo/lib/Config.py

```diff
@@ -186,6 +193,49 @@ def hidden_tags(self):
         hidden_tags = self.cp.get('ls', 'hide_tags')
         return [] if hidden_tags == '' else hidden_tags.split(',')
 
+    def priority_colors(self):
+        """ Returns a dict with priorities as keys and color numbers as value. """
+        pri_colors_str = self.cp.get('colorscheme', 'priority_colors')
+
+        def _str_to_dict(p_string):
+            pri_colors_dict = dict()
+            for pri_color in pri_colors_str.split(','):
```

Comment:
`pri_colors_str` should be `p_string`.

Corresponding test-case: use an empty value for `priority_colors`.